### DISCUSSION TITLE
Looking for Fungible Asset on Testnet

### DISCUSSION THREAD

**Discussion by anthonyjoeseph** [aptos-labs/aptos-developer-discussions#192] [Questions] [answered]
### Discord user ID

snorky_the_dragon

### Describe your question in detail.

I'm trying to build an app that uses the aptos sdk to transfer [layerzero usdc](https://aptoscan.com/coin/0xf22bede237a07e121b56d91a491eb7bcdfd1f5907926a9e58338f964a01b17fa::asset::USDC) between accounts. To start developing, I need some kind of testnet usdc. Does that exit? Is there a faucet for it?

Barring that, is there some other fungible asset that exists on aptos testnet? Ideally with a faucet?

I've already built an app that transfers the native `0x1::aptos_coin`, looking for something other than that for testing.

Am I best off just creating my own with the ["Your First Fungible Asset" tutorial](https://aptos.dev/tutorials/your-first-fungible-asset)?

### What have you tried or looked at?  Or how can we reproduce the error?

The ['coinlist' on aptoscan's testnet](https://aptoscan.com/coins?network=testnet) is empty - maybe this is an error? I imagine that'd be the first place to look for something

Saw this [FACoin tutorial](https://aptos.dev/tutorials/your-first-fungible-asset), wondered if maybe this has some kind of canonical address, and/or if there was some kind of existing faucet for it?

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by kshitijc1506** [comment, marked as answer]
I'm not aware of any official lzUSDC on testnet. 

It should be fairly straightforward to use your own FA to play around with. You can use the FA [move-examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/fungible_asset) to help you customize as well. Just publish the package to testnet, mint to various accounts/fungible stores, and begin testing!

**Reply by anthonyjoeseph** [reply]
Thanks! Going to implement this before marking this as the answer 👍🏻

**Reply by kshitijc1506** [reply]
Sounds good, also might be worthwhile to check if Econia runs a faucet.

**Reply by MoonShiesty** [reply]
we have a faucet for the mUSD FA available on testnet here: https://alphanet.mirage.money/trade

@daorren you can view example testnet mUSD FA transactions on the explorer:

https://explorer.aptoslabs.com/object/0xb87a58d382cf13ad3a744aded0966e39335fd3595004c05fdef58e55a6ee9a09/resources?network=testnet

**Comment by daorren** [comment]
@anthonyjoeseph hi there, I'm trying to check some fungible asset txs, 
I see this discussions today, and I'm wondering that have you created your FA?
